webpack.config.js

```diff
@@ -17,5 +17,5 @@ module.exports = merge(common, {
     new webpack.NamedModulesPlugin(),
     new webpack.HotModuleReplacementPlugin(),
   ],
-  devtool: 'eval-source-map',
+  devtool: 'cheap-module-source-map',
```

Comment:
What's this change for?

Reply:
https://github.com/readmeio/api-explorer/commit/d412bc2680979f71b133f2e1978c6e6fc08a3307